# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/.cargo/config.in b/.cargo/config.in
--- a/.cargo/config.in
+++ b/.cargo/config.in
@@ -42,19 +42,18 @@ git = "https://github.com/mozilla/applic
 replace-with = "vendored-sources"
 rev = "8a576fbe79199fa8664f64285524017f74ebcc5f"
 
 [source."https://github.com/mozilla-spidermonkey/jsparagus"]
 git = "https://github.com/mozilla-spidermonkey/jsparagus"
 replace-with = "vendored-sources"
 rev = "2e56bb9bae5d8211137980a717ee991cc4a5eb98"
 
-[source."https://github.com/kvark/spirv_cross"]
-branch = "wgpu5"
-git = "https://github.com/kvark/spirv_cross"
+[source."https://github.com/kvark/dummy-web"]
+git = "https://github.com/kvark/dummy-web"
 replace-with = "vendored-sources"
 
 [source."https://github.com/kinetiknz/mio-named-pipes"]
 git = "https://github.com/kinetiknz/mio-named-pipes"
 replace-with = "vendored-sources"
 rev = "21c26326f5f45f415c49eac4ba5bc41a2f961321"
 
 [source."https://github.com/kinetiknz/ashmem-rs"]
@@ -77,16 +76,26 @@ git = "https://github.com/hsivonen/chard
 replace-with = "vendored-sources"
 rev = "ed8a4c6f900a90d4dbc1d64b856e61490a1c3570"
 
 [source."https://github.com/hsivonen/chardetng"]
 git = "https://github.com/hsivonen/chardetng"
 replace-with = "vendored-sources"
 rev = "302c995f91f44cf26e77dc4758ad56c3ff0153ad"
 
+[source."https://github.com/gfx-rs/wgpu"]
+git = "https://github.com/gfx-rs/wgpu"
+replace-with = "vendored-sources"
+rev = "d23288e"
+
+[source."https://github.com/gfx-rs/naga"]
+git = "https://github.com/gfx-rs/naga"
+replace-with = "vendored-sources"
+rev = "93db57c"
+
 [source."https://github.com/bytecodealliance/wasmtime"]
 git = "https://github.com/bytecodealliance/wasmtime"
 replace-with = "vendored-sources"
 rev = "824fa69756523f2b6d49029fe25de94130b1f144"
 
 [source.crates-io]
 replace-with = "vendored-sources"
 